packages/reactive-element/src/test/reactive-element_test.ts

```diff
@@ -2294,7 +2294,7 @@ suite('ReactiveElement', () => {
   });
 
   test('can override performUpdate()', async () => {
-    let resolve: (() => void) | undefined;
+    let resolve: ((value?: boolean) => void) | undefined;
```

Comment:
Also never resolved to a `boolean`?

Reply:
Changed to `unknown`. It appears the promise resolver argument is now required.